conda/recipes/ucxx/build.sh

```diff
@@ -3,5 +3,6 @@
 # SPDX-FileCopyrightText: Copyright (c) 2023-2025, NVIDIA CORPORATION & AFFILIATES.
 # SPDX-License-Identifier: BSD-3-Clause
 
-export ucxx_ROOT="$(realpath ./cpp/build)"
+ucxx_ROOT="$(realpath ./cpp/build)"
+export ucxx_ROOT
```

Comment:
this will be removed entirely with the `rattler-build` PR, but it was easier to fix this warning than to leave a TODO to remove an exception from the `.pre-commit-yaml` later